spring-test/src/test/java/org/springframework/test/context/configuration/interfaces/BootstrapWithTestInterface.java

```diff
@@ -34,7 +34,7 @@
 @BootstrapWith(CustomTestContextBootstrapper.class)
 interface BootstrapWithTestInterface {
 
-	static class CustomTestContextBootstrapper extends DefaultTestContextBootstrapper {
+	class CustomTestContextBootstrapper extends DefaultTestContextBootstrapper {
```

Comment:
Every single class modified in the `Removed unnecessary static modifier` commit must be `static`.

Please revert those changes.

Reply:
Isn't that redundant for inner classes of interfaces?

Reply:
It is indeed redundant for nested interfaces, nested enums, and nested annotations, but the same does not apply to classes.

Those static nested classes are not "inner" classes.

In fact, your changes break the build. 

I know it's confusing, but I recommend that you read up on the differences between "static nested" classes and "inner" classes.

An "inner" class is a "nested" class, but a "static nested" class is not an "inner" class.

Reply:
Oh my... I overlooked the fact that you only made those changes to nested types in test interfaces and annotations.

I apologize. You are correct. The static modifier is not required in those.

So please keep those changes in place.